perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/GroupsManagerMethod.java

```diff
@@ -930,7 +930,41 @@ public Vo call(ApiCaller ac, Deserializer parms) throws PerunException {
 	},
 
 	/*#
-	 * Returns all RichGroups containing selected attributes
+	 * Returns list of all RichGroups containing selected attributes as sublist of original list starting at fromIndex
```

Comment:
I would use something like this: 
``Returns sub-list of all RichGroups, each containing selected attributes, starting at fromIndex....``